### DISCUSSION TITLE
What is the difference between this package and @react-native-community/async-storage?

### DISCUSSION THREAD

**Discussion by lowkeyboard** [react-native-async-storage/async-storage#603] [Q&A]
I am aware this one is deprecated and the other not but other than that, is there any particular differences?

**Comment by tido64** [comment]
Please take a look at the discussion here for the full story: https://github.com/react-native-async-storage/async-storage/discussions/507#discussioncomment-222363

tl;dr There is no difference. The package was just renamed.
